test/AspNet.Security.OAuth.Providers.Tests/LinkedIn/LinkedInTests.cs

```diff
@@ -10,7 +10,7 @@ namespace AspNet.Security.OAuth.LinkedIn;
 
 public class LinkedInTests(ITestOutputHelper outputHelper) : OAuthTests<LinkedInAuthenticationOptions>(outputHelper)
 {
-    private Action<LinkedInAuthenticationOptions>? additionalConfiguration;
+    /*private Action<LinkedInAuthenticationOptions>? additionalConfiguration;*/
```

Comment:
Delete redundant code like this, don't leave it lying around in commented-out blocks.